js/index.js

```diff
@@ -40,3 +40,60 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navContent = document.getElementsByTagName("a");
+navContent[0].textContent = siteContent.nav["nav-item-1"];
+navContent[1].textContent = siteContent.nav["nav-item-2"];
+navContent[2].textContent = siteContent.nav["nav-item-3"];
+navContent[3].textContent = siteContent.nav["nav-item-4"];
+navContent[4].textContent = siteContent.nav["nav-item-5"];
+navContent[5].textContent = siteContent.nav["nav-item-6"];
+
+let ctaImgID = document.getElementById("cta-img")
+//console.log(ctaImgID)
+ctaImgID.src = "img/header-img.png";
+
+let ctaText = document.getElementsByClassName("cta-text")
+
+//console.log(ctaText)
+
+ctaText[0].children[0].innerHTML = siteContent.cta["h1"];
+ctaText[0].children[1].textContent = siteContent.cta["button"];
+
+let textContentStuff = document.querySelectorAll(".text-content")
+
+textContentStuff[0].children[0].textContent = siteContent["main-content"]["features-h4"]
+textContentStuff[0].children[1].textContent = siteContent["main-content"]["features-content"]
+textContentStuff[1].children[0].textContent = siteContent["main-content"]["about-h4"]
+textContentStuff[1].children[1].textContent = siteContent["main-content"]["about-content"]
+textContentStuff[2].children[0].textContent = siteContent["main-content"]["services-h4"]
+textContentStuff[2].children[1].textContent = siteContent["main-content"]["services-content"]
+textContentStuff[3].children[0].textContent = siteContent["main-content"]["product-h4"]
+textContentStuff[3].children[1].textContent = siteContent["main-content"]["product-content"]
+textContentStuff[4].children[0].textContent = siteContent["main-content"]["vision-h4"]
+textContentStuff[4].children[1].textContent = siteContent["main-content"]["vision-content"]
+
+let contactText = document.querySelectorAll(".contact")
```

Comment:
Generally, if you not going to mess with a variable any further (e.g. _You're not going to reassign `contactText` to be something else_), it's probably a good idea to use `const`. It will _help_ prevent accidental changes to the variable, and it will communicate to developers that the variable isn't meant to be changed.